attending/index.html

```diff
@@ -43,6 +35,16 @@ <h3 class='space-bottom1'>Public Transport Map</h3>
 
         <p><a href="https://www.rnv-online.de/media/rnv-online.de/Fahrtinfo/Liniennetzplaene/regulaer/Liniennetzplan_HD_regulaer.pdf">See official PDF version</a></p>
 
+        <h3 class='space-bottom1'>Kombiticket – Your Conference Ticket is a Public Transport Ticket</h3>
+        <p>You have a local transport ticket included in your SotM ticket for the days of the State of the Map conference (21–23 September). You will get the ticket by email in advance. Please print it out and take it with you while travelling. It is only valid together with your ID card (EU/Schengen citizens) or passport. It is valid in the whole VRN area, see the map below for details.</p>
+        <div id="vrn_map" style="height: 350px"></div>
+        <script src="vrn_map.js"></script>
+
+        <h3 class='space-bottom1'>Buying Public Transport Ticket</h3>
+        <p>If you want to buy public transport tickets for the days before or after the conference, you might use the vending machines found at most stations. When using busses you can also buy tickets directly from the bus driver. When travelling within Heidelberg, you most likely need to buy a ticket at "Preisstufe 2" (also called “Großwabe Heidelberg”), which costs € 2.60 for a single ticket or € 7.00 for a day ticket. Prices for the day fare get cheaper if you travel in groups of up to 6 people. If you want to buy your ticket with your smartphone, you can use the <a href="https://play.google.com/store/apps/details?id=de.hafas.android.db">Deutsche Bahn app “Navigator”</a> that sells not only train tickets, but also local transport tickets at many places in Germany.</p>
+
+        <p>If you are travelling from / to Heidelberg by train, check out if you have a +City on your DB train ticket. That means that your train ticket includes a <a href="https://www.bahn.de/en/view/offers/germany/tickets-for-local-transport.shtml">City-Ticket</a> and is therefore also valid for a single ride in Heidelberg to your final destination.</p>
```

Comment:
Please add "The city ticket option is valid on the first day of the validity of the ticket only."

This means that it is not valid after 3am if one arrives at night.

This is not mentioned on the linked page but part of section 3.5 of the Beförderungsbedingungen Personenverkehr ([current version](https://www.bahn.de/p/view/mdb/bahnintern/agb/gesamt_2019/mdb_298620_befrderungsbedingungen_der_db_ag_-_stand_16_09_2019.pdf))

Reply:
Technically right. Didn't want to overstrain the foreign readers. l Added a sentence, hope it is as you wished :)